chromium_src/content/common/user_agent.cc

```diff
@@ -8,6 +8,16 @@
 #define BRAVE_GET_ANDROID_OS_INFO \
   include_android_model = IncludeAndroidModel::Exclude;
 
+namespace content {
+std::string BuildModelInfo() {
+  std::string model;
+  return model;
```

Comment:
`return std::string();`

Reply:
Fixed in https://github.com/brave/brave-core/pull/11932/commits/2b48bd0b8e8e77ddb595fafb3f406b51e51d54a5